<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanModuleQTClass</class>
 <widget class="QMainWindow" name="ScanModuleQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanModuleQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CapBtn">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>720</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CAPTURE</string>
    </property>
   </widget>
   <widget class="QFrame" name="Viewer_cad_3D">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>891</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="Viewer_cad_2D">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>891</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="NextBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>720</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>221</width>
      <height>671</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>219</width>
       <height>669</height>
      </rect>
     </property>
     <widget class="QSlider" name="Median3D_slid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Hybrid2D_slid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ParameterTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Parameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Median 3D(Kernal Size)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hybrid Median 2D(On/OFF)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gausian smooth(Std/Rad)</string>
      </property>
     </widget>
     <widget class="QSlider" name="GausianStd_slid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="MeshSmooth_slid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh Smoothing</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter2</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter2</string>
      </property>
     </widget>
     <widget class="QSlider" name="GausianRad_slid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1334</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scanmoduleqt.qrc"/>
 </resources>
 <connections/>
</ui>
